traffic_monitor/tests/.env

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
`.env` should be a symlink to the `.env` file at the top of the repo, which contains a `GO_VERSION` environment variable.